cottontaildb_client/cottontaildb_cli.py

```diff
@@ -125,11 +130,45 @@ def entity(client, args):
     command = args.subcommand
     if command == 'drop':
         response = client.drop_entity(args.schema_name, args.entity_name)
-        print(response)
     elif command == 'about':
         response = client.get_entity_details(args.schema_name, args.entity_name)
-        print(response)
-
+    elif command == 'preview':
+        response = client.preview_entity(args.schema_name, args.entity_name, args.limit)
+    format(response)
+
+def format(obj):
+    """Formats the response object for printing.
```

Comment:
I find this doc string mildly misleading, as the function not only formats the response, but also directly prints it. I suggest either separating concerns and having this function return a string to be printed separately (preferred) or renaming it directly to something like `print_formatted_response`.